### PR TITLE
[KIP-226] Consensus liquidity for Kaia

### PR DESCRIPTION
**Pull request by hyeonLewis**
## Proposed changes

This KIP introduces the consensus liquidity for Kaia.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] KIP Proposal
- [ ] KIP Improvement

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] Used the suggested template: https://github.com/kaiachain/KIPs/blob/main/kip-template.md
- [ ] I have read the [CLA](https://gist.github.com/kaiachain-dev/bbf65cc330275c057463c4c94ce787a6) and signed by comment ```I have read the CLA Document and I hereby sign the CLA``` in first time contribution
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you proposed and what alternatives you have considered, etc.

### CODE REVIEW

**Comment by hyeonLewis** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by ian0371 — COMMENTED** [review]
Please consider these things to be added later:
- CL is orthogonal to PD
- What changes from KIP-82 (is proposer reward distributed to CL? where does the remainder go?)

File: KIPs/kip-xxx.md [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+
+## Simple Summary
+
+Introduce a consensus liquidity mechanism for Kaia. The staked KAIA through the consensus liquidity will contribute to the consensus and [on-chain governance](https://kips.kaia.io/KIPs/kip-81) of Kaia.
+
+## Abstract
+
+This KIP introduces a consensus liquidity mechanism where KAIA and CL Token holders can participate in Kaia network through liquidity provision. Users stake KAIA tokens in the native decentralized exchange (CLDEX), simultaneously providing market liquidity and securing the network. Stakers earn both swap fees and block rewards, creating an efficient capital utilization.
+
+Only existing Kaia validators can participate in the consensus liquidity, which means maintaining 5M KAIA staked solely through `CnStaking` contract is required.
```
[the remark this answers is not in the document]
Comment (hyeonLewis) [reply KIPs/kip-xxx.md]: Thanks!

File: KIPs/kip-xxx.md [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+
+**Block Reward Commission**:
+
+- Commissions can be collected from block rewards
+
+For both commission types, the following parameters are configurable:
+
+- Commission receiver addresses
+- Split ratios for each receiver address
+- Burn ratios
```
[the remark this answers is not in the document]
Comment (hyeonLewis) [reply KIPs/kip-xxx.md:198]: What do you mean above?

[the remark this answers is not in the document]
Comment (ian0371) [reply KIPs/kip-xxx.md:198]: Right above a few lines there's this paragraph:

```
Both commission types have configurable parameters:

- Commission receiver addresses
- Split ratios
- Burn ratios
```

[the remark this answers is not in the document]
Comment (hyeonLewis) [reply KIPs/kip-xxx.md:198]: Ah, thanks. they're the same. Reflected in [de0e4e5](https://github.com/kaiachain/kips/pull/26/commits/de0e4e57be8c6eeaae1274f9eb53f5c2c683a38f)

[the remark this answers is not in the document]
Comment (ian0371) [reply KIPs/kip-xxx.md:198]: It looks like a different commit

[the remark this answers is not in the document]
Comment (hyeonLewis) [reply KIPs/kip-xxx.md:192]: I pushed the commit.

**Review by ian0371 — APPROVED** [review]

**Review by blukat29 — APPROVED** [review]

**Review by hyunsooda — APPROVED** [review]

**Pull request merged**